packages/api/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -48,6 +50,15 @@ export async function sandboxGetDocRefsAndUpsert({
   const { log } = Util.out(`sandboxGetDocRefsAndUpsert - M patient ${patient.id}`);
   const { id: patientId, cxId } = patient;
 
+  await setDocQueryProgress({
```

Comment:
Shouldn't there be a global status update we need to remove somewhere? line 95 maybe.